.github/workflows/unit_tests.sh

```diff
@@ -74,7 +86,7 @@ for phase in "${PHASES[@]}"; do
             # during debugging, wonderful), so let's at least keep a workaround
             # here to make the builds stable for the time being.
             (set +x; while :; do echo -ne "\n[WATCHDOG] $(date)\n"; sleep 30; done) &
-            meson test --timeout-multiplier=3 -C build --print-errorlogs
+            unshare -n bash -c 'ip link set dev lo up; meson test -C build --print-errorlogs'
```

Comment:
I don't think `timeout-multiplier` should be removed here. The sanitizers slow the code down in various (sometimes mysterious) ways. I think instead of increasing the timeouts in `meson.build` `timeout-multiplier` should be used instead here and when the code is built with `-Db_coverage`.

Reply:
ah good catch, that was a copypasta, didn't intend to do it, fixed